### PR TITLE
fix: add userSelect css props for non chrome browsers

### PR DESCRIPTION
**Pull request by malangfox**
## Details
This applies proper user-select styles for browsers that `user-select: none` is not available

### CODE REVIEW

**Review by WoodNeck — APPROVED** [review]
LGTM

File: packages/axes/src/const.ts [hunk 1]
```diff
@@ -75,6 +75,8 @@ export const TRANSFORM = (() => {
 })();
 
 export const PREVENT_DRAG_CSSPROPS = {
+  "-webkit-user-select": "none", /* Safari */
+  "-ms-user-select": "none", /* IE 10 and IE 11 */
```
Comment (daybrush) [inline packages/axes/src/const.ts:79]: It is judged that there will be few users, but how about adding the `-moz-` prefix as well?

**Review by daybrush — APPROVED** [review]
LGTM

File: packages/axes/src/const.ts (hunk 1 above)
Comment (malangfox) [reply packages/axes/src/const.ts:79]: Considering the old version of Firefox, adding `user-select` with `-moz-` prefix should be better.

**Pull request merged**